lms/djangoapps/teams/views.py

```diff
@@ -403,6 +407,28 @@ def post(self, request):
             return Response(CourseTeamSerializer(team).data)
 
 
+class IsEnrolledOrIsStaff(permissions.BasePermission):
```

Comment:
Thanks for fixing this! To clarify, this is in response to @nedbat's comments on https://github.com/edx/edx-platform/pull/9323